testexec/testexec.go

```diff
@@ -161,21 +163,40 @@ func (tcfg Tester) test(t *testing.T, data *testmark.DirEnt, allowExec, allowScr
 	sequenceHunk, sequenceMode := data.Children["sequence"]
 	scriptHunk, scriptMode := data.Children["script"]
 	if !sequenceMode && !scriptMode {
-		return
+		t.Logf("dir %q does not contain a 'script' or 'sequence' hunk", data.Path)
+		if tcfg.DisableStrictMode {
+			t.SkipNow()
+		}
+		t.FailNow()
```

Comment:
Hm, pretty minor thing, but on re-review I notice these four lines recur a number of times pretty much verbatim.  I think extracting a tiny `skipOrFailStrictly(t)` function might be worth it.

The reason to log for it could even be moved in as a fmt string arg and varargs, although I'm neutral on whether or not it's done that way.

Reply:
Adding the logs seems good.